### ISSUE TITLE
Can you achieve training loss of -1000 nats

### ISSUE THREAD

**Issue by leinxx** [hardmaru/write-rnn-tensorflow#36]
Hi, thanks for the nice work. In the paper, the training loss can go down to -1000. I can only get to -5. I wonder how your training loss curve looks like.

Thanks
